tools/autograd/derivatives.yaml

```diff
@@ -247,10 +237,10 @@
 - name: gather(Tensor self, int64_t dim, Tensor index)
   self: grad.type().zeros(self.sizes()).scatter_add_(dim, index, grad)
 
-- name: ge(Tensor self, Scalar other)
+- name: ge_(Tensor self, Scalar other)
```

Comment:
I'm not a fan of file-level comments that describe what changed since a while ago. They're only relevant to a few people (pretty much the people on this PR) and only for a little while. They're better placed in commit messages.

Everything in Declarations.yaml is bound in `VariableType.cpp` with the exception of `_out` variants, which will be added soon.

`should_generate_python_binding` decides what has Python-level bindings. Some functions are manually excluded by `SKIP_PYTHON_BINDINGS`. We currently don't Python generate bindings for namespace-only (i.e. `at::`) functions, but that will change soon.

`derivatives.yaml` already describes binding of derivatives. It doesn't describe Python bindings or whether a function is overridden in VariableType.cpp because it doesn't affect those things.